.github/workflows/integration-tests.yml

```diff
@@ -155,7 +155,7 @@ jobs:
     runs-on: ${{ matrix.runner }}
     timeout-minutes: 30
     env:
-      CMAKE_BUILD_TYPE: "MIN_SIZE_REL"
+      CMAKE_BUILD_TYPE: "Debug"
```

Comment:
Just to over-communicate, I would not expect "Debug" or "MinSizeRel" is the fastest build; I'd expect -O1 is probably faster than both.

But also seeing as Debug seems to be the same speed as Release (15m) maybe something else is causing the slowness...